README.md

```diff
@@ -30,9 +30,8 @@ What does it do?
 | CPython 3.7     | ✅ |   | ✅  | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ |
 | CPython 3.8     | ✅ |   | ✅  | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ |
 | CPython 3.9     | ✅ | ✅ | ✅  | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ |
-| PyPy 2.7 v7.3.3 | ✅ |    |    | ✅  | ✅ |    |    |    |   |
-| PyPy 3.6 v7.3.3 | ✅ |    |    | ✅  | ✅ |    |    |    |   |
-| PyPy 3.7 (beta) v7.3.3 | ✅ | |  | ✅  | ✅ |    |    |   |    |
+| PyPy 2.7 v7.3.4 | ✅ |    |    |    |    | ✅ |    |    |   |
+| PyPy 3.7 v7.3.4 | ✅ |    |    | ✅  |    | ✅ |    |    |   |
```

Comment:
```suggestion
| PyPy 3.7 v7.3.4 | ✅ |    | ✅ |    |    | ✅ |    |    |   |
```
I think this is in the wrong column?

Reply:
How do you mean? That's changing from 32-bit Windows (4th column) to 64-bit Windows (3rd column) for PyPy, right? Or am I miscounting?

Reply:
If I'm counting right, the PR puts the checkmark in the 32-bit column, so my suggestion above is to put it in the 64-bit column (since 7.3.4 is win64 only).

Reply:
Actually, the manylinux column might be wrong as well, so maybe the following is right?
```suggestion
| PyPy 3.7 v7.3.4 | ✅ |    | ✅  |    | ✅ |    |    |   |    |
```
You can see the rendered table here, showing that both checkmarks are in the 32-bit columns: https://github.com/joerick/cibuildwheel/blob/46d81290903d0c23d7e74e487e9478ea04d21425/README.md

Reply:
Oh, wait, I was looking at your suggestion! Seems I move the wrong one (manylinux instead of Windows)!

Thanks for catching!

Reply:
Should both be fixed, now; thanks! :-)